### PR TITLE
add new sponsors to sv2015

### PR DESCRIPTION
**Pull request by irabinovitch**
add new sponsors to sv2015

### CODE REVIEW

File: site/content/events/2015-siliconvalley/_sponsors.txt [hunk 1]
```diff
@@ -11,16 +11,20 @@ end
...
- { :image => 'sysdig.png', :name => 'Sysdig', :link => 'http://www.sysdig.com', :width => '100px', :height => '100px' },
+ { :image => 'chef.png', :name => 'Chef', :link => 'http://www.chef.io', :width => '100px', :height => '100px' },
  { :image => 'cloudpassage.png', :name => 'CloudPassage', :link => 'http://www.cloudpassage.com', :width => '100px', :height => '100px' },
- { :image => 'neptune.png', :name => 'Neptune', :link => 'http://www.neptune.io', :width => '100px', :height => '100px' },
+ { :image => "datadog.png", :name => "Datadog", :link => "http://www.datadoghq.com", :width => '100px', :height => '100px' },
  { :image => 'delphix.png', :name => 'Delphix', :link => 'http://www.delphix.com', :width => '100px', :height => '100px' },
- { :image => 'chef.png', :name => 'Chef', :link => 'http://www.chef.io', :width => '100px', :height => '100px' },
- { :image => 'signalfx.jpg', :name => 'SignalFx', :link => 'https://signalfx.com/', :width => '100px', :height => '100px' },
+ { :image => "docker.png", :name => "Jut", :link => "http://www.docker.io", :width => '100px', :height => '100px' },
```
Comment (bridgetkromhout) [inline site/content/events/2015-siliconvalley/_sponsors.txt]: Are you sure you want to list Docker as Jut? I think this might be a copypasta error.

**Comment by phrawzty** [comment]
The logo for Netuitive (`netuitive.png`) is not like the others - bug or feature? :smile:

**Comment by irabinovitch** [comment]
It's what they provided. It has been up for a few weeks now.  Only change there is alphabetizing the sponsors.

**Comment by phrawzty** [comment]
> It's what they provided. It has been up for a few weeks now. Only change there is alphabetizing the sponsors.

Strange that cb9d176677a27c4869cee095093cef99c82e4a24 apparently includes it, then. :confused:

For hygiene purposes, I'd appreciate it if you could take a look at the commits in this PR and just make sure that it's clean - no accidental inclusions, duplicate commits, etc. From there, either rebase this PR or submit a new one, as necessary. :mag: 

If you need any help with that, just let me know!

**Comment by bridgetkromhout** [comment]
It's probably fine.

**Pull request merged**